EDI/UserInterface/02a_Connections.md

```diff
@@ -0,0 +1,121 @@
+
+- [x]     
+    Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all connections in the list are selected.
+
+
+- [SYNCHRONIZE]  
+    Click this button to synchronize the selected connection. This button is only displayed if a single checkbox in the list of connections is selected. A confirmation message is displayed.
+
+- [DISABLE]  
+    Click this button to disable the selected connection(s). This button is only displayed if the checkbox of at least one active connection is selected.
+
+- [ENABLE]  
+    Click this button to enable the selected connection(s). This button is only displayed if the checkbox of at least one inactive connection is selected. 
+
+- ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit)  
+    Click this button to edit the selected *DataHub Exporter* connection. This button is only displayed if a single checkbox in the list of connections is selected. Alternatively, you can directly click a row in the list to edit a connection. The *Edit connection* view is displayed, see [Edit DataHub exporter connection](#edit-datahub-exporter-connection).
+
+The following functions and fields are available in this view:
+
+- *Name*  
+    Connection name.
+
+- *Status*  
+    Connection status. The following statuses are available:
+    - ![Status](../../Assets/Icons/Status01.png "[Status]") **Active**   
+        The connection is enabled, and data is being synchronized via the connection.
+    - ![Status](../../Assets/Icons/Status04.png "[Status]") **Inactive**   
+        The connection is disabled, and no data is being synchronized via the connection.   
+
+- *Driver*  
+    Driver name.
+
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)  
+    Click this button to add a *DataHub Exporter* connection. The *Create connection* view is displayed, see below.
+
+
+
+## Create DataHub Exporter connection
+
+*DataHub Exporter > Settings > Tab CONNECTIONS > Button Add*
+
+![Create connection](../../Assets/Screenshots/EDI/UserInterface/CreateConnection.png "[Create connection]")
+
+**Connection**   
+The *Connection* view contains the fields required to define a connection. 
+
+- *Name*   
+    Enter a connection name.
+
+- ![Apply](../../Assets/Icons/Check.png "[Apply]") (Apply)  
+    Click this button to apply the entered connection name. This button is only displayed if the connection name has not yet been confirmed. 
+
+-  ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit)  
+    Click this button to edit the connection name. This button is only displayed if the connection name has been confirmed. 
+
+- *Driver*  
+    Click the drop-down list and select the *DataHub Exporter* driver. All installed drivers are displayed.  
+    
+- [SAVE]     
+   Click this button to save the connection. 
+
+
+**Credentials**
+
+This section is only displayed for drivers with further credentials. The fields displayed in the *Credentials* section vary depending on the selected driver.
+
```

Comment:
Sind die Felder der Credentials relevant für die Connection zu EDI? Sollten wir sie nicht kurz beschreiben? Oder ändern sich  je nach dem Treiber? 

Sonst ist die Beschreibung sehr ähnlich wie andere Connections, also in der Zukunft auch hier Verweis auf allgemeine Verbindungen-UI.

Reply:
HÄngt vom Treiber ab